controller/main.go

```diff
@@ -81,32 +83,32 @@ func (c *controller) SetBalancer(l log.Logger, name string, svcRo *v1.Service, _
 		st, svc = svc.Status, svcRo.DeepCopy()
 		svc.Status = st
 		if err := c.client.UpdateStatus(svc); err != nil {
-			l.Log("op", "updateServiceStatus", "error", err, "msg", "failed to update service status")
+			level.Error(l).Log("op", "updateServiceStatus", "error", err, "msg", "failed to update service status")
 			return k8s.SyncStateError
 		}
 	}
-	l.Log("event", "serviceUpdated", "msg", "updated service object")
+	level.Info(l).Log("event", "serviceUpdated", "msg", "updated service object")
```

Comment:
Unless we were to expand this to give more info (what service was updated? how was it updated?), i'd probably keep this as debug to match the other messages it's related to.

Reply:
Maybe we should file an issue to go back through log messages to improve them to include which resource is being looked at / acted upon?